scala/lambdas/dynamo-formatters/src/test/scala/uk/gov/nationalarchives/dynamoformatters/DynamoFormattersTest.scala

```diff
@@ -529,7 +530,6 @@ class DynamoFormattersTest extends AnyFlatSpec with TableDrivenPropertyChecks wi
     resultMap(transferCompleteDatetime).s() should equal("2023-06-01T00:00Z")
     resultMap(upstreamSystem).s() should equal(upstreamSystem)
     resultMap(digitalAssetSource).s() should equal(digitalAssetSource)
-    resultMap(digitalAssetSubtype).s() should equal(digitalAssetSubtype)
     resultMap(originalFiles).ss().asScala.toList should equal(List(originalFilesUuid.toString))
     resultMap(originalMetadataFiles).ss().asScala.toList should equal(List(originalMetadataFilesUuid.toString))
     resultMap(ingestedPreservica).s() should equal("true")
```

Comment:
There's a List on line 537 of the optional parameters which should be missing. Can you add digitalAssetSubtype to that list please.

Reply:
I've been doing some head scratching. This forall construct does not seem to fail the test even when one of the optional values is supplied. Something seem to be syntactically incorrect.
e.g. In this test, if I pass in some value `Option("digital asset subtype")` then the resultMap gets that value but the `resultMap.contains should be(false)` does not fail